templates/config/controller/deployment.yaml.tpl

```diff
@@ -0,0 +1,38 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    control-plane: controller
+  name: ack-system
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ack-{{ .ServiceAlias}}-controller
+  namespace: ack-system
+  labels:
+    control-plane: controller
+spec:
+  selector:
+    matchLabels:
+      control-plane: controller
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        control-plane: controller
+    spec:
+      containers:
+      - command:
+        - ./bin/controller
+        args:
+        image: controller:latest
```

Comment:
Should this be pointing to an image in ECR ?

Reply:
It actually gets replaced via `kustomize edit set image=` in the `kind-build-test.sh` script.